duplicate_url_discarder/_addon.py

```diff
@@ -14,3 +30,9 @@ def update_settings(self, settings: BaseSettings) -> None:
             current_fpr,
             "addon",
         )
+        _setdefault(
+            settings,
+            "ITEM_PIPELINES",
+            "duplicate_url_discarder.DuplicateUrlDiscarderPipeline",
```

Comment:
`_setdefault` expects a class, not an import path.

Reply:
Missed that. Thanks for spotting! https://github.com/zytedata/duplicate-url-discarder/pull/17/commits/20b224a7683ccedf26abefdb13fa5a440cac952e